features/step_six.feature

```diff
@@ -7,19 +7,25 @@ Feature: Step six page
 
   Scenario: Displays step number
     Then I should see step 6 of 20
-  
+
   Scenario: Displays header
     Then I should see 'Do you receive any of the following benefits?' header
 
   Scenario: Displays benefits list
     Then I should see the benefits list:
```

Comment:
Maybe have one scenario for displays instruction copy:
'If you are receiving one of the benefits listed in column 1, select ‘Yes’.                                                
There are some benefits with similar names listed in column 2. If you are receiving one of these benefits, select 'No'.   
If you are not receiving any of benefits listed on this page, select 'No'. 
Another scenario for benefits table.'